### PR TITLE
Completed task 2

### PR DESCRIPTION
**Pull request by LW95x**

### CODE REVIEW

File: __tests__/integration.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+describe("GET /api/topics", () => {
+  test("responds with a 200 status code", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("returns an array of topic objects with 'slug' and 'description' properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
```
Comment (popatre) [inline __tests__/integration.test.js:27]: Your responses should be returned in an object with a key for some context about the data 

I think case, i'd probably expect `topics`

File: __tests__/integration.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        expect(body).toMatchObject([
```
Comment (popatre) [inline __tests__/integration.test.js]: Might be better for other tests to do a forEach and check the shape of the objects in the array - will save you abit of time

https://github.com/northcoders/be-can-you-handle-this/blob/main/__tests__/index.test.js

File: db/controllers/getTopics.controller.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const { findAllTopics } = require("../models/getTopics.model")
+
+exports.getAllTopics = (req, res) => {
+    findAllTopics().then( (data) => {
+        res.status(200).send(data);
```
Comment (popatre) [inline db/controllers/getTopics.controller.js]: need to send these back in an object with a key i.e. {topics:data}

**Review by popatre — CHANGES_REQUESTED** [review]
Just needs a little change to how you send back data in the controller - which will mean the test needs a little change.

I've sent you a link to the tests from Wednesday's sprint. 

Theres a suggestion on using a forEach for the array ones which will save a bit of time on future tests. 

Could also benefit from a 404 bad path test - but you could do this on any ticket as once you've set it up, it works for any bad path

File: __tests__/integration.test.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+describe("GET /api/topics", () => {
+  test("responds with a 200 status code", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("returns an array of topic objects with 'slug' and 'description' properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
```
Comment (ghost) [inline __tests__/integration.test.js:27]: If you want to get a little clever here, you could further destruct in here with ```{body:{topics}}``` then you can access topics directly.

**Review by ghost — APPROVED** [review]
Nice, go ahead and merge.

**Pull request merged**